packages/compass-aggregations/src/modules/explain.ts

```diff
@@ -74,7 +74,7 @@ export const INITIAL_STATE: State = {
   isModalOpen: false,
 };
 
-const reducer: Reducer<State, Actions> = (state = INITIAL_STATE, action) => {
+const reducer: Reducer<State, AnyAction> = (state = INITIAL_STATE, action) => {
```

Comment:
Applies to a few places in this patch: Redux recommendation is not to use type unions for actions because these will not be true (all actions are passed through all reducers, they are not scoped) and makes it easy to miss issues in some cases. See [this](https://redux.js.org/usage/usage-with-typescript#avoid-action-type-unions) and [this](https://phryneas.de/redux-typescript-no-discriminating-union)

Reply:
Ah. @alenakhineika I gave some bad feedback on one of those recent prs. Thanks for the links here, learned new things today.